player/core/src/main/java/dev/jdtech/jellyfin/utils/DownloadUtilities.kt

```diff
@@ -110,12 +109,13 @@ fun isItemAvailable(itemId: UUID): Boolean {
 fun canRetryDownload(itemId: UUID, downloadDatabaseDao: DownloadDatabaseDao, context: Context): Boolean {
     if (isItemAvailable(itemId))
         return false
-    val downloadId = downloadDatabaseDao.loadItem(itemId)?.downloadId ?: return false
+    val downloadId = downloadDatabaseDao.loadItem(itemId)?.downloadId ?: return true
```

Comment:
Is this intended? Does not look quite right to retry if the item is to in the database.